pytest_fixtures/api_fixtures.py

```diff
@@ -8,6 +8,7 @@
 from wrapanapi import GoogleCloudSystem
 
 from robottelo import ssh
+from robottelo.cli.ansible import Ansible
```

Comment:
Introducing robottelo.**cli** into **api**_fixtures?

@jyejare as architect of `pytest_fixtures` directory, could you confirm if that is OK, and guide what should be done in this case?

Reply:
Anyways the `api_fixtures` would be going away soon and will be replaced with component wise / satellite-managements main components wise. 

By the time I would suggest you to put this in component wise fixture file. So In ur case, create ansible_fixture module in fixtures directory and put this fixtures there.

Reply:
@jyejare updated the pr. Created different component fixtures.